src/api.js

```diff
@@ -0,0 +1,27 @@
+export async function getProfile() {
+  const response = await fetch(
+    `https://bootcamp-api.codeit.kr/api/sample/user`,
+    {
+      method: 'GET',
+    }
+  );
+  if (!response.ok) {
```

Comment:
response 자체가 없을 때를 대비해서 ? 붙여주면 좋을 거 같아요! (undefined.ok 가 될 수 있어요)
```suggestion
  if (!response?.ok) {
```